### ISSUE TITLE
/steal notification for police 

### ISSUE THREAD

**Issue by TrooperJoe** [ChainedDev/CityLife#469]
**Is your feature request related to a problem? Please describe.**

The Problem is there is way too many robberies around the city. 
**Describe the feature you'd like**
I would like for the cops to get some kind of call over the dispatch when someone does the /steal command 

**Describe alternatives you've considered**
Calling out for help like the system for EMS and other calls. Or if anyone has any better ideas to slow down the robberies in the city

**Comment by JooniJunebug** [comment]
This is probably the best idea ive seen, please make this happen.

**Comment by FalseH28** [comment]
Call the police and give a description?

**Comment by MatthewHall-Basic** [comment]
Not a good idea at all man, think if I took a hostage to a remote location and then ran his pockets. what the cops would just  randomly show up. You should just hope someone calls the police or you get to keep your phone to call them

**Comment by Sturdyinnit** [comment]
A notification wouldn't really be realistic wouldn't it. Maybe only have it ping if there's an npc nearby (kind of like how the ems help signal works) as if the npc calls the police to report a robbery.

**Comment by REDDGLARE** [comment]
Considering people rob others without any reason other than profit on the regular I think this is a pretty good idea.

**Comment by TrooperJoe** [comment]
> A notification wouldn't really be realistic wouldn't it. Maybe only have it ping if there's an npc nearby (kind of like how the ems help signal works) as if the npc calls the police to report a robbery.

That is why I said in the suggestion "Calling out for help like the system for EMS and other calls. "

**Comment by TrooperJoe** [comment]
> Not a good idea at all man, think if I took a hostage to a remote location and then ran his pockets. what the cops would just randomly show up. You should just hope someone calls the police or you get to keep your phone to call police.

As i said "Calling out for help like the system for EMS and other calls. " and the way that works is if an NPC sees it happening then we get a call. I'm not sure how this would work though

**Comment by Wally-W** [comment]
Yeah, maybe just have successful /steal commands used on another player send out the EMS-style alert to nearby NPC's, if any are close enough to report it then it triggers the Dispatch alert. Not much sense in having a 2-cop requirement to rob people when the chances of being seen mid-robbery is practically non-existent, and for those that say "just report it to the police", when has that ever worked out? PD can probably count the number of times on their fingers lol. 

If Karen sees you pointing a gun at someone with their hands up, and you going that person's pockets, she'd probably call the cops. We're in a huge city and most people are out there robbing people in broad daylight in very public places where there would be plenty of witnesses and probably cameras as well. It's not realistic at all, this isn't Brazil, maybe this will also help reduce those low-effort robberies.

**Comment by gamerdude23** [comment]
kinda bad idea. unless theres a script that makes it so if theres a FOV range for NPCs, then its a bad idea. If you are in the middle of no where and get robbed, police just show up? so if there can be an NPC FOV range that detects stealing, then i wouldn't like it.

**Comment by ChainedDev** [comment]
Yeah would only be added in if the notification depended on a local being nearby

**Comment by ChainedDev** [comment]
It would have to be a small range also so it's not too over powered

**Comment by TrooperJoe** [comment]
This is still here
